bin/compile

```diff
@@ -140,13 +140,49 @@ create_build_env
 [ ! "$NPM_CONFIG_CACHE" ] && NPM_CONFIG_CACHE=$(mktemp -d -t npmcache.XXXXX)
 export YARN_CACHE_FOLDER NPM_CONFIG_CACHE
 
+### Configure vendored package manager
+export YARN2_PATH
+
+if [[ "$YARN2" == "true" ]]; then
+  if [[ "$NODE_MODULES_CACHE" == "true" ]]; then
+    warn "
+      WARNING- You are using Yarn 2. The buildpack won't cache, because Yarn 2 does not provide node modules.
```

Comment:
It does provide a `node_modules` folder with the `node-modules` linker https://yarnpkg.com/configuration/yarnrc#nodeLinker

Though it would be faster to cache the actual cache folder which you can get using `yarn config get cacheFolder`

Reply:
This warning message should be reworded - so thanks for pointing this out. However, we're not supporting this yet in this first phase.